tracing-mock/src/event.rs

```diff
@@ -20,6 +52,18 @@ pub fn msg(message: impl fmt::Display) -> ExpectedEvent {
 }
 
 impl ExpectedEvent {
+    /// Sets the expected name to match an event.
+    ///
+    /// By default an event's name takes takes the form:
+    /// `event <file>:<line>` where `<file>` and `<line>` refer to the
+    /// location in the source code where the event was generated.
+    ///
+    /// To overwrite the name of an event, it has to be constructed
+    /// directly instead of using one of the available macros.
+    ///
+    /// In general, there are not many use cases for expecting an
+    /// event, as the value includes the file name and line number,
+    /// which can make it quite a fragile check.
```

Comment:
maybe something like:
```suggestion
    /// In general, there are not many use cases for expecting an
    /// event with a particular name, as the value includes the file
    /// name and line number. Assertions about event names are
    /// therefore quite fragile, since they will change as the source
    /// code is modified.
```
(in particular, we shouldn't say "not many use-cases for expecting an event" --- there are lots of use-cases for expecting an event, but fewer for expecting an event's name. i think that was probably a typo :))

Reply:
Yep, that was a typo, good catch.